src/azure/cis-1.3.1/rules/azure-cis-1.3.1-2.13.ts

```diff
@@ -76,14 +76,14 @@ export default {
   resource: 'queryazureSecurityContact[*]',
   severity: 'high',
   conditions: {
-    or: [
+    and: [
       {
         path: '@.name',
```

Comment:
if we are only ever checking the default name, could we not exclude the others? Or is it important to show those "pass"

Reply:
It's true, it's not needed. We can use "exclude" here as well. thanks!